src/ResourceManager/Storage/Stack/Commands.Management.Storage/Commands.Management.Storage.csproj

```diff
@@ -82,49 +82,55 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\ARM.Storage.3\ARMStorageProvider.cs">
+    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\ARM.Storage.6\ARMStorageProvider.cs">
       <Link>Common\ARMStorageProvider.cs</Link>
     </Compile>
-    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\ARM.Storage.3\ARMStorageService.cs">
+    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\ARM.Storage.6\ARMStorageService.cs">
       <Link>Common\ARMStorageService.cs</Link>
     </Compile>
-    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\WindowsAzure.Storage.6\AzureContextAdapterExtensions.cs">
+    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\WindowsAzure.Storage.8\AzureContextAdapterExtensions.cs">
       <Link>Common\AzureContextAdapterExtensions.cs</Link>
     </Compile>
-    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\WindowsAzure.Storage.6\AzureStorageContext.cs">
+    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\WindowsAzure.Storage.8\AzureStorageContext.cs">
       <Link>Common\AzureStorageContext.cs</Link>
     </Compile>
-    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\WindowsAzure.Storage.6\LazyAzureStorageContext.cs">
+    <Compile Include="..\..\..\..\Common\Commands.Common.Storage\Adapters\WindowsAzure.Storage.8\LazyAzureStorageContext.cs">
```

Comment:
Updating the linked file references here, since we move on to 8.6.0 version of windowsazure.storage